geps/gep-1897/index.md

```diff
@@ -5,19 +5,16 @@
 
 ## TLDR
 
-This document specifically addresses the topic of conveying HTTPS from the Gateway
-dataplane to the backend (backend TLS termination), and intends to satisfy the single
-use case “As a client implementation of Gateway API, I need to know how to connect to
-a backend pod that has its own certificate”. TLS configuration can be a nebulous topic,
-so in order to drive resolution this GEP focuses only on this single piece of functionality.
+This document aims to address the use case of origination of TLS to backends from implementations.
+This is accomplished by a new BackendTLSPolicy API.
 
-Furthermore, for Gateway API to handle the case where the service or backend owner is doing their own TLS, _and_
-the service or backend owner wants to validate the clients connecting to it, two things need to happen:
+This API solves the question "As an implementation, when I establish a connection to a backend, should I originate a TLS connection? If so, with what properties?"
```

Comment:
A backend is what is on the receiving side of the Gateway as client.   Can you come up with a different term than "backend" for a service that is not the targeted endpoint of a Gateway?